openapi/2023-12.yaml

```diff
@@ -255,6 +255,32 @@ paths:
           $ref: "#/components/responses/HttpError"
         "5XX":
           $ref: "#/components/responses/HttpError"
+  /events:
+    post:
+      summary: Receive an event
+      tags:
+        - Event
+      operationId: postEvent
+      security:
+        - ApiKeyAuth: [ ]
+      description: |
+        Receive an event from the merchant's store. It can be used to trigger flows in Recart.
```

Comment:
Similar to the above, we should use a statement that describe the usage from the user's perspective.